types/newman/index.d.ts

```diff
@@ -22,6 +23,8 @@ export interface NewmanRunOptions {
     envVar?: Collection | CollectionDefinition | string | undefined;
     /** A globals JSON / file path for the current collection run. */
     globals?: VariableScope | VariableScopeDefinition | string | undefined;
+    /** An override to global variables.  See: https://github.com/postmanlabs/newman/blob/develop/lib/run/options.js */
+    globalVar?: Collection | CollectionDefinition | string | undefined;
```

Comment:
I'm don't think this can be a string. I also don't think it's supposed to be a Collection or CollectionDefinition. This looks like it should really be `{ key: string, value: string } | { key: string, value: string }[] | undefined`.

The same comments apply for envVar. I may well be wrong on both these points, just going based on the docs and example usages.

Reply:
You're definitely right! Wow I misread the code. I agree with your types and I have pushed and update.